src/create-template-orders.js

```diff
@@ -266,6 +252,9 @@ function _generateTemplateOrderImportDraft(
     nextReminderDateISOString = nextDeliveryCronDate.toISOString()
   }
   const templateOrder = {
+    paymentInfo: {
```

Comment:
I think, it would be better to set paymentInfo later on to object, because now it looks like it's always empty 🤔

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/19/commits/bcd13739dd1c89678a16ab671a2c4daf46523eb5